src/EnergyPlus/PollutionModule.cc

```diff
@@ -246,7 +246,7 @@ void GetPollutionFactorInput(EnergyPlusData &state)
         }
 
         // If Steam Conversion Efficiency defined by the User is negative or zero then a default of 25% will be assigned.
-        if (state.dataIPShortCut->rNumericArgs(1) > 0.0) {
+        if (state.dataIPShortCut->rNumericArgs(3) > 0.0) {
```

Comment:
I've never looked at this module before. Sacre bleu! It has about (at least) 10X more code than it needs to. This entire logic can be implemented using loops over fuels and pollutants. Instead it has copies of the code (with one parameter changed) for each fuel and pollutant and sometimes for the cross product of each loop and pollutant. Not sure who wants to take this on, but this will probably be the most satisfying couple of hours you will spend with EnergyPlus source code.

Reply:
@amirroth yes, it was painful and I had similar thoughts. Probably would be very satisfying work. Can we make it a new defect? I’m willing to take a whack at it.

Reply:
@RKStrand, sure. Go for it!